packages/@o3r/core/schematics/component/presenter/index.ts

```diff
@@ -184,11 +204,11 @@ export function ngGenerateComponentPresenter(options: NgGenerateComponentSchemat
       ]), MergeStrategy.Overwrite));
     }
 
-    return chain(rules)(tree, context);
+    return chain(rules);
```

Comment:
why do you remove the reference to the tree? If you chain this after another schematic, are the changes done in your previous schematic applied?

Reply:
I changed it to `async` and got trouble with typings if I keep it.
We can check together the use case that you have in mind.
Because mine seems to work